warmrestart/warm_restart.cpp

```diff
@@ -96,6 +97,34 @@ bool WarmStart::checkWarmStart(const std::string &app_name, const std::string &d
     return true;
 }
 
+/*
+ * Get warmStartTimer for an application in a docker (default to be swss)
+ * return value 0 = OK,  -1 = failure
+ * timer value is saved to timer_value if succeed. timer value 0 is invalid.
+ */
+int WarmStart::getWarmStartTimer(uint32_t &timer_value, const std::string &app_name,
+    const std::string &docker_name)
+{
+    auto& warmStart = getInstance();
+    std::string timer_name = app_name + "_timer";
+    std::string timer_value_str;
+
+    warmStart.m_cfgWarmRestartTable->hget(docker_name, timer_name, timer_value_str);
+
+    unsigned long int temp_value = strtoul(timer_value_str.c_str(), NULL, 0);
+    if (temp_value != 0 && temp_value != LONG_MAX)
+    {
+        timer_value = (uint32_t)temp_value;
+        return 0;
+    }
+    else
+    {
+        SWSS_LOG_WARN("Error in getting warmStartTimer for docker: %s, app: %s",
+                docker_name.c_str(), app_name.c_str());
```

Comment:
If the log level is warning,  Is there some code logic to put the default timer value in DB?